softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java

```diff
@@ -9,7 +9,7 @@
  */
 
 @Component
-public class HandlerFactory<T extends Handler> extends AbstractFactory<T> {
+public class HandlerFactory<T extends RequestHandler> extends AbstractFactory<T> {
```

Comment:
AbstractFactory can work with handlers and with routers
![abstract](https://cloud.githubusercontent.com/assets/16840046/23401942/39ec0f8c-fdb2-11e6-9bdf-3b157f4030b0.png)